buildcache

```diff
@@ -49,20 +49,29 @@ environment_variables_to_skip = [
     '.*SESSION_COOKIE.*',
     'BUILDCACHE.*',
     'COLUMNS',
+    'COLORTERM',
     'OLDPWD',
     'DISPLAY',
+    'DBUS.*',
+    'GPG_AGENT_INFO',
+    'GNOME.*',
     'LM_LICENSE_FILE',
     'LOGNAME',
     'MAIL',
     'MAKELEVEL',
     'MFLAGS',
     'MAKEFLAGS',
+    'NXSESSIONID',
     'SHLVL',
     'TERM',
     'TERMCAP',
+    'TMUX.*',
+    'SESSION_MANAGER',
+    'SHADOW_XAUTHORITY',
     'SSH.*',
     'USER',
     'WINDOW',
+    'XAUTHORITY',
     'XDG*'
 ]
```

Comment:
Thanks for adding these.

I'm thinking I should modify fpgamake write command scripts rather than using environment variables, and then add an option to buildcache to ignore the environment, except for PATH and PWD to avoid false misses.